c7n/resources/ebs.py

```diff
@@ -1236,7 +1237,7 @@ def create_encrypted_volume(self, ec2, v, key_id, instance_id):
                 {'Key': 'maid-crypt-remediation', 'Value': instance_id},
                 {'Key': 'maid-origin-volume', 'Value': v['VolumeId']},
                 {'Key': 'maid-instance-device',
-                 'Value': v['Attachments'][0]['Device']}])
+                 'Value': v['Attachments'][0]['Device']}] + unencrypted_volume_tags)
 
```

Comment:
lgtm, nit, we might need to trim this to only the last 50 tags due to tag cardinality limits, but that seems low probability to me.

Reply:
You're right, but personally I prefer getting an error message regarding the tags, rather than losing them :)